packages/vitest/src/node/cli/cli-config.ts

```diff
@@ -540,11 +540,7 @@ export const cliOptionsConfig: VitestCLIOptions = {
     description: 'Enable cache',
     argument: '', // allow only boolean
     subcommands: {
-      dir: {
-        description: 'Path to the cache directory',
-        argument: '<path>',
-        normalize: true,
-      },
+      dir: null,
```

Comment:
Maybe it would be better to add a handler that throws an error? I think CLI will still propagate the value even if it's not defined in a schema

Reply:
Yeah I noticed that, I'll updated it later